package.json

```diff
@@ -1,80 +1,81 @@
 {
-    "name": "ghubber",
-    "version": "0.9.0",
-    "private": true,
-    "scripts": {
-        "start": "node node_modules/react-native/local-cli/cli.js start --reset-cache",
-        "start:ios": "concurrently -r 'npm run start' 'npm run ios && google-chrome http://localhost:8081/debugger-ui'",
-        "start:android": "concurrently -r 'npm run start' 'npm run android && google-chrome http://localhost:8081/debugger-ui'",
-        "ios": "node node_modules/react-native/local-cli/cli.js run-ios",
-        "android": "node node_modules/react-native/local-cli/cli.js run-android",
-        "flow": "flow check",
-        "test": "jest",
-        "lint": "eslint --cache .",
-        "lint:staged": "staged-files '**/*.js' -- $(npm bin)/eslint",
-        "postversion": "react-native-version"
-    },
-    "pre-commit": {
-        "run": "lint:staged"
-    },
-    "dependencies": {
-        "babel-plugin-module-resolver": "^2.7.1",
-        "babel-plugin-transform-define": "^1.3.0",
-        "babel-plugin-transform-remove-console": "^6.8.5",
-        "base-64": "^0.1.0",
-        "commonmark": "^0.28.1",
-        "commonmark-react-renderer": "^4.3.3",
-        "github-flow-js": "^0.18.2",
-        "i18n-js": "^3.0.1",
-        "lodash": "^4.17.4",
-        "moment": "^2.18.1",
-        "query-string": "^5.0.0",
-        "react": "16.0.0-alpha.12",
-        "react-native": "^0.48.4",
-        "react-native-device-info": "^0.11.0",
-        "react-native-htmlview": "^0.12.0",
-        "react-native-sentry": "^0.25.0",
-        "react-native-side-menu": "^1.1.3",
-        "react-native-swiper": "^1.5.12",
-        "react-native-vector-icons": "^4.4.0",
-        "react-navigation": "^1.0.0-beta.12",
-        "react-redux": "^5.0.6",
-        "redux": "^3.7.2",
-        "redux-logger": "^3.0.6",
-        "redux-thunk": "^2.2.0",
-        "rn-viewpager": "^1.2.4"
-    },
-    "devDependencies": {
-        "babel-eslint": "^8.0.1",
-        "babel-jest": "21.2.0",
-        "babel-preset-react-native": "4.0.0",
-        "concurrently": "^3.5.0",
-        "eslint": "^4.8.0",
-        "eslint-plugin-flowtype": "^2.37.0",
-        "eslint-plugin-react": "^7.4.0",
-        "eslint-plugin-react-native": "^3.1.0",
-        "flow-bin": "^0.56.0",
-        "jest": "21.2.1",
-        "pre-commit": "^1.2.2",
-        "react-native-version": "^2.3.1",
-        "react-test-renderer": "16.0.0-alpha.12",
-        "sentry-cli-binary": "^1.20.0",
-        "staged-files": "^0.1.2"
-    },
-    "jest": {
-        "preset": "react-native"
-    },
-    "contributes": {
-        "languages": [
-            {
-                "id": "flow",
-                "aliases": [
-                    "Flow"
-                ],
-                "extensions": [
-                    ".js"
-                ]
-            }
+  "name": "ghubber",
```

Comment:
Please revert ident for this file
Thanks!

Reply:
I'm not sure what to revert here?

Reply:
I mean, not needed to change this file (code indentation - spaces)